find_nav_points_service_server:
  ros__parameters:
    distance_to_shelf_for_find_pose: 1.10
    resolution_for_find_nav_rad: 0.1
    angle_min_rad: -2.5
    angle_max_rad: 2.5
    cant_casileros_verify: 2                                              